chronoline/chronoline.js

```diff
@@ -543,6 +549,23 @@ function Chronoline(domElement, events, options) {
 
                 elem.attr('title', myEvent.title);
 
+                if (myEvent.attrs.text) {
+                    try {
+                        var tNodeVal = typeof (myEvent.attrs.text) === "string" ? myEvent.attrs.text : myEvent.attrs.text.value;
+                        var yOffset = myEvent.attrs.text.offset || 10;
+                        var tNode = t.paper.text(startX, upperY + yOffset, tNodeVal);
+                        if (myEvent.attrs.text.options) {
```

Comment:
this is a good idea to allow customization. can you leave a comment at line 170 noting that you can add these options to customize?